docs_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build !urfave_cli_no_docs
-// +build !urfave_cli_no_docs
+//go:build to_fix
+// +build to_fix
```

Comment:
@meatballhat i need your help to fix this

Reply:
It looks to me like all of the test diffs are around bool flags having usage text with `(default: false)`. If that's an intentional change, then I think what's needed is for the `testdata/expected-*` files to be updated accordingly.

Reply:
Ok thanks let me take a look.